addons/csi-vmware-cloud-director/base/vcloud-basic-auth.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vcloud-basic-auth
+  namespace: kube-system
+data:
+  username: "{{ with .Credentials.VCD_USER }}{{ . | b64enc }}{{ end }}"
+  password: "{{ with .Credentials.VCD_PASSWORD }}{{ . | b64enc }}{{ end }}"
+  refreshToken: "{{ with .Credentials.VCD_API_TOKEN }}{{ . | b64enc }}{{ end }}"
```

Comment:
Either username/password or refreshToken should exist in this secret; we should drop fields that are not used instead of setting them to empty strings. We should retain the existing implementation for this https://github.com/kubermatic/kubeone/blob/main/addons/csi-vmware-cloud-director/vcloud-basic-auth.yaml

Reply:
done